package.json

```diff
@@ -37,7 +37,8 @@
       "ts"
     ],
     "files": [
-      "src/*.test.ts"
+      "src/*.test.ts",
```

Comment:
The `src/**/*.test.ts` should already cover the `src/*.test.ts` so we should be able to remove this line and just use the one with the `/**/`.